### DISCUSSION TITLE
Add the player settings menu item the Players side bar

### DISCUSSION THREAD

**Discussion by sh-fr** [music-assistant/support#1299] [Feature requests and ideas]
Hi,
Would be nice and convenient if the player settings were available in Players sidebar (displayed when clicking on the speaker icon at the bottom right).  For each player there is already the volume, a settings cog could be displayed underneath to take you to the respective player's settings.
Awesome  job on the latest Beta 35!

**Comment by OzGav** [comment]
The UI is being completely reworked so keep an eye out for that in a few months

**Reply by OzGav** [reply]
Beta 30 has a reworked player list. The settings for the player is available from the overflow menu 
![image](https://github.com/user-attachments/assets/2d541e6e-6d5d-40d9-a284-c5cf8d3a3687)
